packages/stylelint-plugin/src/rules/no-focus-visible/no-focus-visible.test.ts

```diff
@@ -0,0 +1,69 @@
+import rule from './no-focus-visible';
+import * as stylelint from 'stylelint';
+
+describe('no-focus-visible', () => {
+  it('should fail with :focus-visible', async () => {
+    const { errored, results } = await stylelint.lint({
+      code: `.foo:focus-visible { color: red; }`,
```

Comment:
Let's add a fixture with `.foo :focus-visible` to ensure that space is handled, too.